ui/app/styles/components/linked-block.scss

```diff
@@ -1,16 +0,0 @@
-.linked-block {
```

Comment:
Could have sworn I did `git mv`. Is there a good way to fix that after the fact?

Reply:
Hmm usually GH is good about picking that up, but I wouldn't worry about it.